corehq/apps/userreports/views.py

```diff
@@ -385,7 +385,7 @@ def import_report(request, domain):
 def report_source_json(request, domain, report_id):
     config = get_document_or_404(ReportConfiguration, domain, report_id)
     config._doc.pop('_rev', None)
-    return json_response(config)
+    return json_response(config._doc)
```

Comment:
(it's a lazy translation - which `json_response` handles in the json encoder)

it does seem curious that a translation would end up in an `jsonobject` though.